src/components/Carousel/DotsSlider/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+//Types
+
+import { props } from "./types";
+
+// Styled Components
+
+import { StyledDot, StyledUl } from "./styles";
+
+const DotsSlider: React.FC<props> = ({ amountOfDots, click, activeTabIndex }: props) => {
+    const dotsJsx = [];
+
+    for (let i = 0; i < amountOfDots; i++) {
```

Comment:
Aca se podria utilizar un map para reducir la cantidad de codigo. Ej: https://es.reactjs.org/docs/lists-and-keys.html#rendering-multiple-components

Reply:
Coincido con @9gustin